packages/tldraw/src/state/data/filesystem.ts

```diff
@@ -97,18 +95,15 @@ export async function openFromFileSystem(): Promise<null | {
 
   const fileHandle = blob.handle ?? null
 
-  await saveFileHandle(fileHandle)
+  await saveFileHandle(fileHandle as FileSystemFileHandle | null)
```

Comment:
This works without a cast
```suggestion
  await saveFileHandle(fileHandle)
```